CONTRIBUTING_JS.md

```diff
@@ -135,6 +159,18 @@ A `gh-pages` branch will be created, and this should be published to via the Git
 
 This makes API docs available, and the types are linked through to from other modules published by aegir.
 
+aegir can also update the `About` section of a README.md based on the `@packageDocumentation` of your entry file.
```

Comment:
```suggestion
aegir can also update the `About` section of a README.md based on the `@packageDocumentation` of your entry file via the `aegir docs` command.
```

Reply:
Thank you! I was trying to remember the exact command and couldn't.. I figured it would get caught :)